pkg/asset/machines/openstack/machines_test.go

```diff
@@ -0,0 +1,170 @@
+package openstack_test
+
+import (
+	"fmt"
+	"testing"
+
+	machineapi "github.com/openshift/cluster-api/pkg/apis/machine/v1beta1"
+	"github.com/openshift/installer/pkg/asset/machines/openstack"
+	"github.com/openshift/installer/pkg/types"
+	openstackTypes "github.com/openshift/installer/pkg/types/openstack"
+)
+
+func newConfig() *types.InstallConfig {
+	return &types.InstallConfig{
+		AdditionalTrustBundle: "",
+		SSHKey:                "",
+		BaseDomain:            "basedomain.example.com",
+		Networking:            nil,
+		ControlPlane:          nil,
+		Compute:               nil,
+		Platform: types.Platform{
+			OpenStack: &openstackTypes.Platform{
+				Region:                 "RegionOne",
```

Comment:
Region is deprecated along with TrunkSupport and OctaviaSupport

Reply:
oh I missed that, thanks! If it's OK for you, I'll comment out `Region` in a follow-up; there's a couple more tests I want to add but didn't want to clutter this urgent PR.